docs/source/api/python/nn-pytorch.rst

```diff
@@ -145,3 +145,18 @@ Network Embedding Modules
 
     ~dgl.nn.pytorch.DeepWalk
     ~dgl.nn.pytorch.MetaPath2Vec
+
+Utility Modules for Graph Transformer
+----------------------------------------
+.. autosummary::
+    :toctree: ../../generated/
+    :nosignatures:
+    :template: classtemplate.rst
+
```

Comment:
Is it necessary to have a `gt` namespace? Is this settled from a prior discussion?

Reply:
We previously decided to move gt-related modules to a separate folder for better management. Besides, in my opinion, maintaining a separate namespace avoids a file too large and gives clear guidance for contributors.